src/rcu.rs

```diff
@@ -22,7 +22,7 @@ thread_local! {
     // odd value means the current thread is about to access the active_epoch of an Rcu
     // a thread has a single epoch counter for all Rcu it accesses,
     // as a thread can only access one Rcu at a time
-    static THREAD_EPOCH_COUNTER: OnceCell<Arc<AtomicU8>> = OnceCell::new();
+    static THREAD_EPOCH_COUNTER: OnceCell<Arc<AtomicU8>> = const { OnceCell::new() };
```

Comment:
Ah, I just noticed in the `thread_local!` docs: 

> This macro supports a special `const {}` syntax that can be used
> when the initialization expression can be evaluated as a constant.
> This can enable a more efficient thread local implementation that
> can avoid lazy initialization.

So, I think this change makes sense.